openapi_to_fastapi/tests/test_ihan_standards.py

```diff
@@ -133,3 +135,29 @@ def test_auth_provider_header_is_missing(tmp_path):
     }
     spec["paths"]["/Company/BasicInfo"]["post"]["parameters"] = [auth_header]
     check_validation_error(tmp_path, spec, ihan.AuthProviderHeaderMissing)
+
+
+def test_missing_html_file(tmp_path):
+    spec = deepcopy(COMPANY_BASIC_INFO)
+    spec_path = tmp_path / "spec.json"
+    spec_path.write_text(json.dumps(spec))
+    (tmp_path / "spec.jsonld").write_text('{"a":"b"}')
+    with pytest.raises(ihan.StandardComponentMissing):
```

Comment:
Should we also test with empty content in the file?

Reply:
added